lib/wda/utils.js

```diff
@@ -150,10 +150,19 @@ CODE_SIGN_IDENTITY = ${signingId}
   return xcconfigPath;
 }
 
+async function getPidUsingAppName (udid, appName) {
```

Comment:
There are two places in this method where we return `null`, and so the calling code will fail. So logging why we're returning it (unparsable stdout, err) would clarify at this step.